src/feeds/music-genre.feed.ts

```diff
@@ -0,0 +1,41 @@
+import { Feed } from '../core/feed';
+import { IgAppModule } from '../types/common.types';
+import { MusicGenreFeedResponseItemsItem, MusicGenreFeedResponseRootObject } from '../responses';
+
+export class MusicGenreFeed extends Feed<MusicGenreFeedResponseRootObject, MusicGenreFeedResponseItemsItem> {
+  protected nextCursor?: string;
```

Comment:
We have `.serialize()` method in `Feed` class
https://github.com/dilame/instagram-private-api/blob/master/src/core/feed.ts#L65
It takes only `@Expose`d properties of feed. So we need to expose all the meaningful properties, like [here](https://github.com/dilame/instagram-private-api/blob/master/src/feeds/direct-inbox.feed.ts), for example